app/models/event.rb

```diff
@@ -2,6 +2,7 @@ class Event < ApplicationRecord
   belongs_to :creator, class_name: 'User', foreign_key: 'user_id'
   has_many :attendances, foreign_key: 'attended_event_id'
   has_many :guests, through: :attendances, source: "attendee"
```

Comment:
[LINTING ERROR]
`"attendee"` should in single quotes `'attendee'`.

Reply:
fixed

Reply:
> The user's page should contain user's created event (you od not have to separate this into past and upcoming) and event User is attending (This you separate into past and upcoming). This would make use of your attended_events relationship and upcoming and past event scopes.
Now the profile shows the events created by the logged in user

Reply:
**Summary of Changes**

1. Change the login to be done with the email

1. Update the testing to support this feature

1. Modify the Home page, not to invite to signup to logged-in users

1. Modify the user's profile to show the user's events as required

1. add the date of the event to the create event form and edit event

1. Create Validation to the Event's date, Cannot create an event in the past, cannot modify a past event, an already created event cannot be rescheduled to a date in the past

1. create the testing for the date validation

1. fix the linter issues